app/views/account/pages/new.js.erb

```diff
@@ -0,0 +1,2 @@
+var updatedLocations = $('#new_page_ajax', $('<%= j render "new" %>')).dialog(modal: true);
+$('#new_page_ajax').replaceWith(updatedLocations);
```

Comment:
`$('#new_page_ajax').html('<%= j render "new" %>').dialog(modal: true);` should make it better